charts/consul/templates/ingress-gateways-deployment.yaml

```
@@ -9,6 +9,16 @@
 {{- $defaults := .Values.ingressGateways.defaults }}
 {{- $names := dict }}
 
+{{- $gateways := .Values.ingressGateways.gateways }}
```

```suggestion
{{- /* Check if gateway names are unique. */ -}}
{{- $gateways := .Values.ingressGateways.gateways }}
```

What do you think about adding a comment to these conditionals (also in tgw)?